man/networkd.conf.xml

```diff
@@ -62,6 +62,15 @@
         If <varname>SpeedMeter=no</varname>, the value is ignored. Defaults to 10sec.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>ManageForeignRoutingPolicyRules=</varname></term>
+        <listitem><para>A boolean. When true, <command>systemd-networkd</command> will store any
+        routing policy rules configured by other tools in its memory, and
+        <command>systemd-networkd</command> will remove rules that are not configured in .network files.
```

Comment:
"`<command>systemd-networkd</command> will store any routing policy rules configured by other tools in its memory`" is an internal detail. Maybe just say:
"`When true, <command>systemd-networkd</command> will remove rules that are not configured in .network files (except for rules with "proto kernel"). When false, it will not remove any foreign rules, keeping them even if they are not not configured in a .network file.
`"